resources/lang/en/cat/backstories.en.json

```
@@ -1,6 +1,13 @@
 {
     "en": {
         "clan_founder": "m_c is one of the founding members of the Clan.",
+        "clan_guide1": "m_c once belonged to another Clan, but felt the pull of c_n and chose to become their guide.",
+        "clan_guide2": "m_c was a distant relative of one of the c_n founders and heard the Clan's call for guidance in those early moons.",
+        "clan_guide3": "m_c spent moons wandering the afterlife before finding the Clan.",
```

```suggestion
        "clan_guide3": "m_c spent moons wandering the afterlife before finding c_n.",
```
Feels a little weird to call it "the Clan".